tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -762,6 +762,7 @@ def process_complete(self, url, payload):
     def process_action(self, url, payload):
         action = payload["action"]
         cmd_id = payload["id"]
+        params = payload["params"]
```

Comment:
Maybe this fallback can be useful
```suggestion
        params = payload["params"] if hasattr(payload, "params") else {}
```

Reply:
it will always have the params attribute, would not it? unless `hasattr(payload, "params")` also checks that it is not None, which I doubt it does?